docs-2.0/graph-computing/algorithm-description.md

```diff
@@ -35,12 +43,21 @@ Parameter descriptions are as follows:
 
 - Nebula Analytics
 
-  |Parameter|Predefined value|Description|
-  |:--|:--|:--|
-  |`ITERATIONS`|`10`| The maximum number of iterations.|
-  |`IS_DIRECTED`|`true`| Whether to consider the direction of the edges. If set to `false`, the system automatically adds the reverse edge.|
-  |`EPS`|`0.0001`| The convergence accuracy. When the difference between the result of two iterations is less than the `EPS` value, the iteration is not continued.|
-  |`DAMPING`|`0.85`| The damping coefficient. It is the jump probability after visiting a page.|
+  - Input parameters
+
+    |Parameter|Predefined value|Description|
+    |:--|:--|:--|
+    |`ITERATIONS`|`10`| The maximum number of iterations.|
+    |`IS_DIRECTED`|`true`| Whether to consider the direction of the edges. If set to `false`, the system automatically adds the reverse edge.|
+    |`EPS`|`0.0001`| The convergence accuracy. When the difference between the result of two iterations is less than the `EPS` value, the iteration is not continued.|
+    |`DAMPING`|`0.85`| The damping coefficient. It is the jump probability after visiting a page.|
+
+  - Output parameters
+
+    |Parameter|Type|Description|
+    |:--|:--|:--|
+    |`VID`|Determined by `vid_type` | The vertex ID.|
+    |`VALUE`|double| The PageRank value of the vertex.|
```

Comment:
vertex or vertices?

Reply:
Vertex. There is only one value in a row.